changelog/fix_fix_rubocop_11164_suppress_server_mode_message.md

```diff
@@ -0,0 +1 @@
+* [#11164](https://github.com/rubocop/rubocop/issues/11164): [Fix rubocop#11164] Suppress server mode message with -f json. ([@jasondoc3][])
```

Comment:
That's just my two cents. Can you remove `[Fix rubocop#11164]` from the entry and add backticks to `enclose `-f json`?

```suggestion
* [#11164](https://github.com/rubocop/rubocop/issues/11164): Suppress server mode message with `-f json`. ([@jasondoc3][])
```